requirements.txt

```
@@ -8,6 +8,7 @@ django-rest-swagger==2.2.0
 django==2.1.5
 djangorestframework==3.7.3
 h5py==2.6.0
+higlass-python==0.1.14
```

Just curious, why do we need higlass python here?
```suggestion
higlass-python==0.2.1
```

---

I added a fallback for retrieving tiles unknown tiles here: https://github.com/higlass/higlass-server/pull/112/files#diff-78e7d7e49c57ab56a37ec8d5315592faR525

This means that if we update `higlass-python` to support new tile types, we can just declare the handler here (https://github.com/higlass/higlass-python/blob/34100b7d02992649d175e4e022dfa1ed353d174d/higlass/tilesets.py#L175), bump the version of `higlass-python` and avoid having to update `higlass-server` separately.

Why is it better to update in `higlass-python` rather than `higlass-server`? Because we can use `higlass-python` in Jupyter but not the other way around.

---

Sounds good. I guess my question is rather where we use higlass-python because I couldn't find any import. Am I missing something?